LanguageFeatures/Unquoted-imports/test.package1/lib/dotted.name1.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @description Library file for testing unquoted imports.
+/// @author sgrekhov22@gmail.com
+
+// SharedOptions=--enable-experiment=unquoted-imports
+
+String libraryId = "dotted.name1.dart";
```

Comment:
Perhaps?:
```suggestion
String libraryId = "dotted.name11";
```

Reply:
Why `dotted.name11`? Changed to `dotted.name1`.

Reply:
I just looked at the other declarations playing a similar role: It looked like the `libraryId` would be created by removing `.dart` from the end of the file name and adding `1`, e.g., `c.d.dart` has `c.d1`.